css/style.css

```diff
@@ -220,34 +220,52 @@ main {
 }
 
 .toggle-switch {
-  display: flex;
+  display: inline-flex;
   align-items: center;
   justify-content: space-around;
   height: 30px;
-  width: 70px;
-  margin-left: 10px;
+  width: 55px;
+  margin-left: 5px;
   background-color: var(--secondary-color);
-  border-radius: 2px;
+  border-radius: 5px;
+  position: relative;
 }
 
-input[type="checkbox"] {
-  -moz-appearance: none;
-  margin: 0;
+.toggle-switch input{
+  display: none;
```

Comment:
This actually might allow you to remove the `span class="toggle"` also if you make the input visible but slide along?